src/collision.cpp

```diff
@@ -25,11 +25,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "gamedef.h"
 #ifndef SERVER
 #include "client/clientenvironment.h"
+#include "client/localplayer.h"
 #endif
 #include "serverenvironment.h"
 #include "serverobject.h"
 #include "util/timetaker.h"
 #include "profiler.h"
+#include <iomanip>
 
 // float error is 10 - 9.96875 = 0.03125
 //#define COLL_ZERO 0.032 // broken unit tests
```

Comment:
`COLL_ZERO` should be removed if it is unused now.

Reply:
Haven't touched any of that, the comment was there before, should I still remove it?

Reply:
Well you have removed some uses of `COLL_ZERO`, so are the any left?

Reply:
removed